packages/frontend-app-api/src/wiring/createApp.tsx

```diff
@@ -287,13 +310,11 @@ export function createApp(options: {
 
     const discoveredFeatures = getAvailableFeatures(config);
     const loadedFeatures = (await options.featureLoader?.({ config })) ?? [];
-    const allFeatures = Array.from(
-      new Set([
-        ...discoveredFeatures,
-        ...(options.features ?? []),
-        ...loadedFeatures,
-      ]),
-    );
+    const allFeatures = deduplicateFeatures([
+      ...discoveredFeatures,
+      ...loadedFeatures,
+      ...(options.features ?? []),
```

Comment:
nice! makes me think that we definitly should document this hierarchy to avoid confusion when the plugin does not look as configured:)

Reply:
Indeed!